model/ListView/ListViewModel.bbj

```diff
@@ -0,0 +1,181 @@
+
```

Comment:
move each Data Class in to its own file.  You even created a separate Folder called ListView in model
so no need to put it all in one file ;)